bundle/terranova/plan.go

```diff
@@ -171,7 +172,12 @@ func CalculatePlanForDeploy(ctx context.Context, b *bundle.Bundle) error {
 				}
 				continue
 			}
-			value, err := structaccess.Get(config, fieldPath)
+			dynPath, err := dyn.NewPathFromString(fieldPath)
+			if err != nil {
```

Comment:
Because the above code uses dynvar, I don't think this ever triggers.

We should be more strict here to begin with and make sure everything after `${` is valid.